tests/unit/app/test_api.py

```diff
@@ -18,7 +18,9 @@ def test_read_root(anon_client):
     assert prev_resp.headers["location"] == "./docs"
 
 
-def test_request_summary_is_logged(caplog):
+def test_request_summary_is_logged(
+    caplog,
+):  # pylint: disable=missing-function-docstring
```

Comment:
wouldn't there be a way to disable it at the module level (`tests/unit/`?)

Reply:
Modified pyproject.toml to do so